tests/test_scheme.py

```diff
@@ -188,3 +346,24 @@ def test_replace_labels(values, labels, new_labels, expected):
         expected,
         check_names=False,
     )
+
+
+def test_replace_labels_misc_table():
+
+    db = audformat.testing.create_db(minimal=True)
+    db['misc'] = audformat.MiscTable(
+        pd.Index(['spk1', 'spk2'], name='speaker')
+    )
+    scheme = audformat.Scheme(labels='misc', dtype='str')
+    db.schemes['scheme'] = scheme
+    db['table'] = audformat.Table(index=audformat.filewise_index(['f1', 'f2']))
+    db['table']['columns'] = audformat.Column(scheme_id='scheme')
+    db['table']['columns'].set(['spk1', 'spk2'])
+
+    db['misc-new'] = audformat.MiscTable(
+        pd.Index(['spk1', 'spk2', 'spk3'], name='speaker')
+    )
+    db.schemes['scheme'].replace_labels('misc-new')
+
+    # Using not assigned scheme
+    scheme.replace_labels('misc-not-assigned')
```

Comment:
What exactly are we testing here?

Reply:
We are testing replacing the scheme with a non-existing misc table.
I updated the table name and added several `assert` statements to the test.